src/main/resources/content/cft/AWS-SNS-subscription-is-not-configured-with-HTTPS.json

```diff
@@ -1 +1,8 @@
-{"severity":"medium","rule":"$.Resources.[*].Type equals AWS::SNS::Subscription and ($.Resources.[*].Properties.Protocol exists and $.Resources.[*].Properties.Protocol equals http)","id":"b53e5177-96e1-4999-a9c8-6400190910bb","enabled":true,"resourceType":"sns","policy":"AWS SNS subscription is not configured with HTTPS"}
\ No newline at end of file
+{
+  "severity": "medium",
+  "rule": "$.Resources.[*].Type equals AWS::SNS::Subscription and ($.Resources.[*].Properties.Protocol exists and $.Resources.[*].Properties.Protocol equals http)",
```

Comment:
The rule is not correct. Pls update it. Right now it will work with any resource having a protocol parameter.